Vwitter/Models/VentAudience.m

```diff
@@ -0,0 +1,34 @@
+//
+//  VentAudience.m
+//  Vwitter
+//
+//  Created by Christina Li on 7/12/22.
+//
+
+#import "VentAudience.h"
+
+@implementation VentAudience
+
+@dynamic ventId;
+@dynamic userId;
+@dynamic groupId;
+
++ (nonnull NSString *)parseClassName {
+    return @"VentAudience";
+}
+
+- (instancetype)initWithVentId:(Vent *)ventId withAudience:(NSObject *)audience {
```

Comment:
important: If you ever find yourself using `NSObject *` parameters, or even worse, `id` parameters (which basically tells the compiler "any object can be passed here"), that would be a red flag.

In this case, having 2 separate initializers would be better, one initializer with a `PFUser` as the parameter and another with `GroupDetails` as the parameter. It is actually pretty frequent that, even if you can accomplish something with 1 method, it is often better to create 2 separate methods/functions (even if it ends up being more code), just for simplicity and safety.

Currently, you could call `[[VentAudience alloc] alloc:@"some_nonsensical_string"]` and the compiler would not be able to protect/warn you because even strings are subclasses of `NSObject`. 

nit: also similar to above, when you are just assigning a pointer to a real object (like a real vent), saying `ventId` might be a bit misleading, so just calling it a `vent` would be better